src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -135,7 +135,7 @@ public void withR2D3(String userAction, final Command command)
          new Dependency[] {
             Dependency.cranPackage("htmltools", "0.3.6"),
             Dependency.cranPackage("htmlwidgets", "1.2", true),
-            Dependency.embeddedPackage("r2d3")
+            Dependency.cranPackage("r2d3", "0.2.0")
```

Comment:
You should add the `true` parameter (as with htmlwidgets) so that we install the source package (compatible with more versions of R and eliminates the problem with lagging binaries on CRAN)

Reply:
Already added by JJ. I was not aware that `true` was needed to install from source, good to know!